### PR TITLE
fix: Update the default themes in app theming

### LINKED ISSUE #13720: [Enhancement]: Update the default themes in app theming
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

The default themes could be improved in app theming as the colors are not either appealing or typically used in apps.

### Why should this be worked on?

By default apps built in appsmith should look amazing

### LINKED ISSUE #13992: [Bug]: Wrong tooltip text in theme property pane
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Wrong tooltip message is shown on hovering the border-radius and box-shadow

### Steps To Reproduce

[LOOM DEMO](https://www.loom.com/share/b55bbe7aa3804e9689f7f4c4c46b9d1f)

### Public Sample App

_No response_

### Version

Cloud

### PR DESCRIPTION
**Pull request by jsartisan**
This PR updates the default themes and adds a few new themes as well.

## Description

Fixes #13720
Fixes #13992 

## Type of change

- Bug fix

## How Has This Been Tested?
- Manually tested

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by somangshu** [comment]
@jsartisan who is reviewing this?

**Comment by jsartisan** [comment]
The change was primarily on the system-themes.json. Rest are just cypress changes required. I'll ask Aishwarya to review this.

File: app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/ThemingTests/Basic_spec.js [hunk 1]
```diff
@@ -497,94 +497,11 @@ describe("App Theming funtionality", function() {
...
-      .click()
-      .parent()
-      .siblings()
-      .find(".t--theme-card > main > section > div > main")
-      .eq(1)
-      .invoke("css", "background-color")
-      .then((backgroudColor) => {
-        expect(backgroudColor).to.eq("rgb(246, 246, 246)");
-      });
-    //#endregion
```
Comment (Aishwarya-U-R) [inline app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/ThemingTests/Basic_spec.js:586]: Why are these themes removed? I see you have replaced current themes to new... so please do the above actions by applying new theme in place of old themes. This case is to verify we are able to apply all featured themes & it cant be removed.

File: app/client/cypress/manual_TestSuite/CommentedScriptFiles/App_Theming_New_spec.js [hunk 2]
```diff
@@ -0,0 +1,902 @@
+// const commonlocators = require("../../../../locators/commonlocators.json");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/manual_TestSuite/CommentedScriptFiles/App_Theming_New_spec.js:1]: This file is deleted, I think you have not pulled latest release. Please do!

Comment (Aishwarya-U-R) [reply app/client/cypress/manual_TestSuite/CommentedScriptFiles/App_Theming_New_spec.js:1]: And I do not see the ts spec files theme updates? And all failing cases are related to that.

**Review by Aishwarya-U-R — APPROVED** [review]
Approving cypress changes!

**Pull request merged**